buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -42,27 +42,27 @@ object Config {
     //   https://detekt.github.io/detekt/performance.html
     //   https://detekt.github.io/detekt/style.html
     // Release info: https://github.com/detekt/detekt/releases
-    const val DETEKT_VERSION = "1.19.0"
+    const val DETEKT_VERSION = "1.21.0"
```

Comment:
@LuisRocketRangel Please run the Android Studio `detekt` Run Configuration to see if any additional violations are found with the latest version of detekt